### DISCUSSION TITLE
Transferring plasma-manager to nix-community

### DISCUSSION THREAD

**Discussion by magnouvean** [nix-community/infra#1320] [General]
Hello :)
I've been a main maintainer for [plasma-manager](https://github.com/pjones/plasma-manager) for some time now and am wondering if it could be possible to move the project over to nix-community? It has for some time been a goal to move it to nix-community, but there's been a wish for the project to mature a bit before going ahead with it, but lately the project has become more and more mature and more and more capable of configuring the various parts of kde plasma, so I think it is mature enough to go ahead with it now. If you have a different opinion, or some special requirements before transferring I am happy to listen to this though and do the work to fulfill the potentially missing requirements in the future.

It's worth noting that I am not the owner of the repository, @pjones is (though he is no longer an active maintainer), so transferring the repo would maybe require @pjones's assistance (I don't know exactly how this works), but I'm sure that's not a problem if this indeed is the case :)

If it were to be moved I am happy to be registered as a maintainer of the repo. Outside of me @toast003 has also been a maintainer of the project, though I haven't heard from there about the status regarding moving forward as a maintainer if the repo were to be moved to nix-community.

**Comment by zowoq** [comment]
AFAIK only the the repo owner can transfer the repo.

@pjones, @magnouvean I've sent you an invite to join @nix-community and created a team @nix-community/plasma-manager.

**Comment by toast003** [comment]
Hi!
I'm onboard with moving plasma manager here, although as @zowoq said we'll need pjones's help to transfer the repo.

I did stop contributing to the project a while ago thanks to burning myself out (among other things) but I'm feeling better now, so if we move the repo I would like to be in the plasma-manager team too

**Comment by pjones** [comment]
The project has been moved to nix-community.

I want to give a big thank you to @magnouvean and @toast003 for all their hard work on this!

**Comment by zowoq** [comment]
@toast003 I've sent you an invite to join https://github.com/nix-community and @nix-community/plasma-manager.
